packaging/images/microshift-aio/Dockerfile

```diff
@@ -46,8 +46,7 @@ COPY $BUILD_PATH/unit /usr/lib/systemd/system/microshift.service
 COPY $BUILD_PATH/kubelet-cgroups.conf /etc/systemd/system.conf.d/kubelet-cgroups.conf
 COPY packaging/crio.conf.d/microshift.conf /etc/crio/crio.conf.d/microshift.conf
 
-# OCP_VERSION pushed ahead to 4.9.11 because aarch64 is now available, and it is backwards compatible
-RUN export OCP_VERSION=4.10.13 && \
+RUN export OCP_VERSION=4.10.15 && \
```

Comment:
I think this should be 4.10.16, or is there a reason to keep it to .15?

Reply:
updated to 4.10.16